src/catalog/catalog.cpp

```diff
@@ -180,6 +181,9 @@ void Catalog::BootstrapSystemCatalogs(storage::Database *database,
   system_catalogs->GetTableCatalog()->InsertTable(
       COLUMN_CATALOG_OID, COLUMN_CATALOG_NAME, CATALOG_SCHEMA_NAME,
       database_oid, pool_.get(), txn);
+  system_catalogs->GetTableCatalog()->InsertTable(
```

Comment:
When we bootstrap catalog tables(e.g pg_table, pg_index, etc..), did you handle the logic to insert corresponding record into pg_layout table?